### ISSUE TITLE
radare2 fails to parse this DOS file

### ISSUE THREAD

**Issue by ITAYC0HEN** [radare/radare2#14438]
### Work environment

| Questions                                            | Answers
|------------------------------------------------------|--------------------
| OS/arch/bits (mandatory)                             | Any
| File format of the file you reverse (mandatory)      | MS-DOS executable
| Architecture/bits of the file (mandatory)            | x86 16-bit
| r2 -v full output, **not truncated** (mandatory)     | radare2 3.7.0-git 22412 @ linux-x86-64 git.3.6.0-14-g4a1392932 commit: 4a1392932e08296283bbd8edb09cc35998a66d29 build: 2019-06-27__21:01:59

### behavior
radare2 fails to detect
 - Entry Point
 - Imports
 - Exports 

In the following screenshot, you can see a function that r2 failed to detect **(and I manually defined**) and that the imported functions that are being called are wrong:

![image](https://user-images.githubusercontent.com/20182642/60289596-722b7f80-991f-11e9-8c56-d1cdf2614c4e.png)

Moreover, all this is wrong:

```
[0000:0000]> ii
[Imports]
Num  Vaddr       Bind      Type Name

[0000:0000]> ir
[Relocations]

0 relocations

[0000:0000]> iE
[Exports]
Num Paddr      Vaddr      Bind     Type Size Name

[0000:0000]> ie
[Entrypoints]
vaddr=0x00000000 paddr=0x00000200 haddr=-1 type=program

1 entrypoints

```

IDA:
![image](https://user-images.githubusercontent.com/20182642/60289544-54f6b100-991f-11e9-9e87-9923ad43012b.png)

### Expected behavior
radare2 should detect all this and show the right information

Binary:

[REVERSI.zip](https://github.com/radare/radare2/files/3336090/REVERSI.zip)

**Comment by GustavoLCR** [comment]
This is a NE file. I dont think radare supports it yet. See #14486

**Comment by ITAYC0HEN** [comment]
@GustavoLCR, I am closing this pr since it is fixed in #14573 
Let me know if you want me to keep it open
